src/main/webapp/app/tasks/default-task.service.ts

```diff
@@ -16,10 +16,18 @@ export class DefaultTaskService implements TaskService {
     return this.http.post<Task>(this.baseUrl + '/tasks', {name} as Task);
   }
 
+  update(id: string): Observable<Task> {
+    return this.http.post<Task>(this.baseUrl + '/tasks', {id} as Task);
```

Comment:
why is the update implemented as post? I guess this should be a put.

Reply:
yes, you are right. Thanks for the feedback 👍